frontend/src/Components/AgencySearch/AgencySearch.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { AgencySearchStyled } from './AgencySearch.styled';
+import { Link, useRouteMatch } from 'react-router-dom';
+
+function AgencySearch(props) {
+  const match = useRouteMatch();
+  return (
+    <AgencySearchStyled data-testid="AgencySearch">
+      <p>AgencySearch</p>
+      <Link to={`${match.url}/80`}>View durm or something</Link>
```

Comment:
Do we still want this filler in here?

Reply:
I think a certain amount of filler is ok. I hope to indicate that there are a lot of things that need changing. Plus that link is the only way to navigate to the AgencyData page as it is.

Reply:
It's fine I was just wondering.